### DISCUSSION TITLE
[9 Settembre 2022] Modifica CPU

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/architettura-degli-elaboratori#71] [Modifiche architettura MIPS (M-Z)]
![Esame](https://github.com/sapienzastudentsnetwork/archelab2223/assets/107944476/4b53286d-57ad-4b71-8d54-f9d784672add)
![Esame](https://github.com/sapienzastudentsnetwork/archelab2223/assets/107944476/96a7e893-a985-409f-8778-fc1c937ba99a)
![CPU](https://github.com/sapienzastudentsnetwork/archelab2223/assets/107944476/917fd33a-1f35-414a-90e7-5a5e9cc6b633)
---
### Soluzione del Prof
Le soluzioni fornite dal professore potrebbero essere incomplete o poco chiare, quindi sono benvenute soluzioni che le completano, oppure soluzioni alternative e/o con spiegazioni più approfondite: 
- [Disegno.pdf](https://github.com/sapienzastudentsnetwork/archelab2223/files/11666160/AE.2021-22.-.Appello.3.-.2022-09-09.-.Esercizio.CPU.-.Diagramma.pdf)
- [Soluzione.pdf](https://github.com/sapienzastudentsnetwork/archelab2223/files/11666171/AE.2021-22.-.Appello.3.-.2022-09-09.-.Esercizio.CPU.-.Scheda.di.calcolo.-.E2.2.E2.3.pdf)

**Comment by daromacarlo** [comment]
Propongo soluzione per punto 4) :
![44](https://github.com/sapienzastudentsnetwork/archelab2223/assets/135441131/f778e99d-f4b9-410c-9591-f2adb9da58af)

**Comment by CiottoloMaggico** [comment]
![immagine](https://github.com/sapienzastudentsnetwork/archelab2223/assets/79638284/b7fa2238-e5a7-459b-8267-c4a6c3e5c00c)
Soluzione analoga al prof e esercizio 4 più nel dettaglio
